samples/nativdlg/dialog1.rc

```diff
@@ -21,7 +21,7 @@
 
 #include "resource.h"
 
-#if !defined(__GNUWIN32__) && !defined(__BORLANDC__)
+#if !defined(__GNUWIN32__)
```

Comment:
I have no idea why are we testing for this one here...

Reply:
All code in the sample looks 20 years old. I don't plan to rewrite it but I can change this one to `#include <windows.h>`, like other .rc files.
I also noticed the label is not shown in the button. I'll fix that as well.
![nativdlg](https://user-images.githubusercontent.com/8088070/96651971-3c8d7f80-1336-11eb-9f86-532ac4e64580.png)

Reply:
This is how it looks like after 1a2d813
![nativdlg-fixed](https://user-images.githubusercontent.com/8088070/96653292-01d91680-1339-11eb-87a7-df73057e321f.png)